apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictor/cell-population-predictor.component.scss

```diff
@@ -0,0 +1,83 @@
+:host {
+  display: block;
+  background: linear-gradient(to bottom, #e6eaf0, #ffffff);
```

Comment:
Move this background gradient into the app component.
Also colors are specified using `var(--sys-***)`.
#e6eaf0 is `var(--sys-surface-dim)` and
#ffffff is `var(--sys-surface-container-lowest)`